pytorch_lightning/trainer/states.py

```diff
@@ -58,7 +58,7 @@ class RunningStage(LightningEnum):
     """
     TRAINING = 'train'
     SANITY_CHECKING = 'sanity_check'
-    VALIDATING = 'validate'
+    VALIDATING = 'val'
```

Comment:
can we use the same key values across RunningStage and TrainerState? why is it `val` here but `validate` above?

Reply:
Yes, good catch ! @carmocca Should we have `val` or `validate` ? I can add a mapping within Flash to map validate to val, but we are adding more inconsistency for validation.

Reply:
It doesn't matter much to use as we use the Enum and not the str

But i'd keep `validate` in `TrainerState` as that is the function name

Now, whether to use `val` or `validate` in `RunningStage`, some people are pushing to standardize it to `val` (https://github.com/PyTorchLightning/pytorch-lightning/issues/6094#issuecomment-782770802). I don't mind either way personally.

Reply:
Reverted back to validate. It should be changed in another PR if needed.